src/python/pants/engine/target.py

```diff
@@ -1902,6 +1925,36 @@ def to_unparsed_address_inputs(self) -> UnparsedAddressInputs:
         return UnparsedAddressInputs(self.value or (), owning_address=self.address)
 
 
+class BinaryTargetDependencies(SpecialCasedDependencies):
```

Comment:
PackageTargetDependencies, given that we renamed `./pants binary` to `./pants package`?

Reply:
Yeah, that's a better name.